streams/tests/test_dask.py

```diff
@@ -54,6 +55,33 @@ def test_map(c, s, a, b):
 
 
 @gen_cluster(client=True)
+def test_compute(c, s, a, b):
+    source = Stream()
+
+    @delayed
+    def inc1(x):
+        """Immediate delayed function"""
+        return inc(x)
+
+    def inc2(x):
+        """Function that just returns a delayed"""
+        return delayed(inc)(x)
+
+    L1 = source.to_dask().compute(inc1).sink_to_list()
+    L2 = source.to_dask().compute(inc2).sink_to_list()
```

Comment:
What is your objective here?  What are you trying to accomplish?  Is this preferable to `.map(inc)` in some way?